hack/build-local-images.py

```diff
@@ -111,6 +111,7 @@
         "tag": "latest",
         "directory": "builder/docker/docker-builder",
         "binaries": {
+            "openshift-docker-build": "/usr/bin/openshift-docker-build",
```

Comment:
/usr/bin/openshift-docker-build is a symlink to oc, so this shouldn't be necessary.